app/response/views.py

```diff
@@ -63,14 +64,26 @@ def response_file(request_id):
         file_options = process_privacy_options(files)
         email_content = flask_request.form['email-content']
         for privacy, files in file_options.items():
-            send_response_email(request_id, privacy, files, email_content)
+            send_file_email(request_id, privacy, files, email_content)
     return redirect(url_for('request.view', request_id=request_id))
 
 
-# TODO: Implement response route for extension
-@response.route('/extension/<request_id>', methods=['GET', 'POST'])
-def response_extension():
-    pass
+@response.route('/extension/<request_id>', methods=['POST'])
+def response_extension(request_id):
+    """
+    Extension response endpoint that takes in the metadata of an extension for a specific request from the frontend.
+    Calls add_extension to process the extension form data.
+
+    :param request_id: Specific FOIL request ID for the extension
+    :return: redirects to view request page as of right now (IN DEVELOPMENT)
+    """
+    extension_data = flask_request.form
+    add_extension(request_id,
```

Comment:
There should be some error handling before calling this function.

Try retrieving the elements and checking if they exist. If they don't, a meaningful error should be returned (shouldn't ever happen in this case, but when we have an API, it might be an issue)